docs/SSH.md

```diff
@@ -1,47 +1,83 @@
+  * [MobaXterm](https://mobaxterm.mobatek.net/) - A feature-rich console commands and drag & drop client that includes most ways to remotely connect to another computer, including SSH, SFTP, and even VNC (allows viewing the screen of another computer).
+* Mac:
+  * [CyberDuck](https://cyberduck.io/?l=en)
 
-1) Pre-boot. On a system with an SD-card reader, access your SD-card and create an empty file called `ssh` or `ssh.txt` in the root directory of the **boot** partition.
 
-2) In **raspi-config**: after booting.
-```
-sudo raspi-config
-```
-**interfacing options >> SSH >> Enable >> reboot your pi**
+# Enable SSH
+
+> **NOTE:** 
+>
+> Starting with RetroPie 4.2, in order to keep the default image secure, [SSH is disabled by default](https://www.raspberrypi.org/blog/a-security-update-for-raspbian-pixel/). You will not be able to remotely connect to it until it has been enabled using **ONE** of the instructions below.
+
+## (Option 1) From the SD Card
+
+If your computer has an SD-card reader or a special USB stick that allows inserting an SD card, plug it into your computer, open the new drive to access your SD-card's `boot` partition,  and create an empty file called `ssh` or `ssh.txt` in the root directory. Ignore any warnings about the drive needing to be repaired.
+
+## (Option 2) From the RetroPie menu
 
-For more information on why SSH was disabled see [HERE](https://www.raspberrypi.org/blog/a-security-update-for-raspbian-pixel/).
+1. Select `raspi-config` from the RetroPie menu after booting up.
+2. Select `Interface Options`
+3. Select `SSH`
+4. Choose `Yes`
+5. Select `Ok`
+6. Choose `Finish`
 
-### When using BerryBoot
+## (Option 3) When using BerryBoot
 
 When using RetroPie with [BerryBoot](http://www.berryterminal.com/doku.php/berryboot) you cannot use `raspi-config`. There is a workaround to enable SSH:
-* Boot to RetroPie (via BerryBoot)
-* Enter shell by pressing <kbd>Ctrl + F4</kbd>
-* `cd /boot`
-* `sudo touch ssh`
-* `sudo reboot now`
-* Done!
+1. Boot to RetroPie (via BerryBoot)
+2. Enter shell by pressing <kbd>Ctrl + F4</kbd>
+3. `cd /boot`
+4. `sudo touch ssh`
+5. `sudo reboot now`
+6. Done!
 
-## Connecting
+# Connecting
 
-When you first start putty you can sign in with the following credentials:
+Four credentials are needed to remotely connect to your Raspberry Pi: IP address, port, username, and password. These will be the same with any client (WinSCP, Putty, MobaXterm, etc).
+
+## Default Login
+In your chosen client, enter the following:
+
+* IP address: **See below note**
+* Port: `22`
+* Username: `pi`   
+* Password: `raspberry`
+
+> **Note:**
+>
+> The IP address is unique to your local network, and can be found by selecting the `Show IP` option in the `RetroPie` menu after booting up your Raspberry Pi.
+
+Example using Putty:
 
 ![putty](https://cloud.githubusercontent.com/assets/10035308/10655671/23eaa6b2-7834-11e5-8c85-9266c5ab808a.png)
```

Comment:
This Putty image isn't loading for me when I preview this, but it does work when I use the Markdown All In One extension for VSCode. Hopefully it works in production.